### PR TITLE
React redux dev

### PR DESCRIPTION
**Pull request by sachTyagi**
Created App container

### CODE REVIEW

File: book-store/.idea/book-store.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/book-store.iml]: Why is this required

File: book-store/.idea/encodings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/encodings.xml]: Why is this required

File: book-store/.idea/encodings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+<project version="4">
```
Comment (varshasaha) [inline book-store/.idea/encodings.xml]: Why is this required

File: book-store/.idea/encodings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  <component name="Encoding">
+    <file url="PROJECT" charset="UTF-8" />
+  </component>
+</project>
```
Comment (varshasaha) [inline book-store/.idea/encodings.xml]: Why is this required

File: book-store/.idea/jsLibraryMappings.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/jsLibraryMappings.xml]: Why is this required

File: book-store/.idea/libraries/book_store_node_modules.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<component name="libraryTable">
```
Comment (varshasaha) [inline book-store/.idea/libraries/book_store_node_modules.xml]: Why is this required

File: book-store/.idea/misc.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/misc.xml]: Why is this required

File: book-store/.idea/modules.xml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/modules.xml]: Why is this required

File: book-store/.idea/workspace.xml [hunk 9]
```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (varshasaha) [inline book-store/.idea/workspace.xml]: Why is this required

**Review by varshasaha — CHANGES_REQUESTED** [review]
Please explain why we need xml?

File: book-store/.idea/book-store.iml (hunk 1 above)
Comment (sachTyagi) [reply book-store/.idea/book-store.iml]: May be my Webstorm editor has created it and I might have pushed it in the git sorry my bad, but I am only following the example, I am not using xml.

Comment (sachTyagi) [reply book-store/.idea/book-store.iml]: I am following the same pattern which is there in the example which you shared. Please look into the files - App.js and index.js
